### PR TITLE
Use Roboto Mono font for amounts

### PR DESCRIPTION
**Pull request by lukaw3d**
| Before | After |
| --- | --- |
| wallet dev oasis io_account_oasis1qrtyn2q78jv6plrmexrsrv4dh89wv4n49udtg2km | localhost_3000_account_oasis1qrtyn2q78jv6plrmexrsrv4dh89wv4n49udtg2km |

### CODE REVIEW

**Comment by buberdds** [comment]
is the "after" screenshot up to date? in a preview URL spacing looks weird 
Screenshot from 2024-07-15 19-04-40

File: src/styles/main.css [hunk 1]
```diff
@@ -20,6 +20,16 @@
...
 
+/** Use Arial for "." and "," instead of Roboto Mono. */
+@font-face {
+  font-family: 'Roboto Mono';
+  font-style: normal;
+  font-weight: 300;
+  font-display: swap;
+  src: local("Arial");
+  unicode-range: U+002E, U+002C;
+}
```
Comment (lukaw3d) [inline src/styles/main.css:31]: > @buberdds is the "after" screenshot up to date? in a preview URL spacing looks weird

It is up to date. I tried it in Firefox now, and it looks like your screenshot. So this code isn't working consistently :shrug:

Comment (lukaw3d) [reply src/styles/main.css]: Updated to use Rubik and it fixes my Firefox
localhost_3000_account_oasis1qrtyn2q78jv6plrmexrsrv4dh89wv4n49udtg2km

Try again?

Comment (buberdds) [reply src/styles/main.css]: looks much better

**Review by buberdds — APPROVED** [review]

**Pull request merged**